templates/partials/introduction/preview.html

```diff
@@ -23,7 +23,6 @@ <h2 class="ons-u-mt-m">{{ intro.title }}</h2>
           "data-ga": "click",
           "data-ga-category": "Preview Survey",
           "data-ga-action": "Open panel",
-          "data-ga-label": question.question
```

Comment:
Don't we still want to send something in the `data-ga-label` field?

Reply:
Didn't add one as wasn't sure it was required and looks like we don't always attach a label in other events? But can add something here if it is needed?

Reply:
I think this is something we need to look into across eQ. We need to be consistent about what we send and what attributes we are setting. I think theres some stuff getting set that isn't even getting used too

Reply:
Yeah there is another ticket at the moment laying the groundwork for this, checkin what we are sending 👍